.github/workflows/perf.yaml

```diff
@@ -0,0 +1,40 @@
+name: Akka Strem Perf Spec
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  test:
+    name: Test
+    runs-on: self-hosted
+    if: github.repository == 'akka/akka'
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Cache Coursier cache
+        uses: coursier/cache-action@v6.2
+
+      - name: Run Performance tests
+        run: |
+          cp .jvmopts-perf-ci .jvmopts
+          sbt \
+            -Dakka.ci-server=true \
```

Comment:
We can remove this property if env var `CI` is defined in the self-hosted runner.

Reply:
removed